dali/operators/reader/loader/file_loader.h

```diff
@@ -129,9 +129,11 @@ class FileLoader : public Loader< CPUBackend, ImageFileWrapper > {
         std::ifstream s(file_list_);
         DALI_ENFORCE(s.is_open(), "Cannot open: " + file_list_);
 
-        string image_file;
-        while (s >> image_file) {
-          images_.push_back(image_file);
+        vector<char> line_buf(16 << 10);  // 16 kB should be more than enough for a line
+        char *line = line_buf.data();
+        while (s.getline(line)) {
```

Comment:
Please use `http://www.cplusplus.com/reference/istream/istream/getline/` and make sure that line_buf is not overflow.